src/cbang/socket/SocketDefaultImpl.cpp

```diff
@@ -209,13 +209,21 @@ void SocketDefaultImpl::setReceiveTimeout(double timeout) {
 
 
 void SocketDefaultImpl::setReceiveBuf(int size) {
+#ifdef __APPLE__
```

Comment:
These are no longer needed.  I took out the MAX_INT defaults.

Reply:
Ok. I removed this.